modules/installation-gcp-enabling-api-services.adoc

```diff
@@ -28,7 +28,7 @@ to complete {product-title} installation.
 .Procedure
 
 * Enable the following required API services in the project that hosts your
-cluster. See
+cluster. You may also enable optional API services which are not required for installation. See
```

Comment:
```suggestion
cluster. You can also enable optional API services, which are not required for installation. See
```
See: https://www.ibm.com/docs/en/ibm-style?topic=word-usage
See: https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=CCSENABLE&title=Commas+in+restrictive+and+nonrestrictive+clauses